Formula/dbus-glib.rb

```diff
@@ -3,6 +3,10 @@ class DbusGlib < Formula
   homepage "https://wiki.freedesktop.org/www/Software/DBusBindings/"
   url "https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.112.tar.gz"
   sha256 "7d550dccdfcd286e33895501829ed971eeb65c614e73aadb4a08aeef719b143a"
+  license all_of: [
+    any_of: ["AFL-2.1", "GPL-2.0-or-later"],
+    "GPL-2.0-or-later", # dbus/dbus-bash-completion-helper.c
```

Comment:
This style only works if `any_of` is last, e.g.
```suggestion
    "GPL-2.0-or-later", # dbus/dbus-bash-completion-helper.c
    any_of: ["AFL-2.1", "GPL-2.0-or-later"],
```

Otherwise, needs braces:
```suggestion
    { any_of: ["AFL-2.1", "GPL-2.0-or-later"] },
    "GPL-2.0-or-later", # dbus/dbus-bash-completion-helper.c
```